<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="seed_spin_box">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>45</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="width_spin_box">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>45</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="height_spin_box">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>170</y>
      <width>45</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="seed_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seed</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameboard_width_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Table width</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameboard_height_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Table height</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_push_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>220</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_push_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>260</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_push_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>300</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QWidget" name="game_widget" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>109</y>
      <width>461</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphics_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="timer_text_browser">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>380</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="score_text_browser">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>470</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>340</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="score_lable">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>440</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Score</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="game_info_text_browser">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 34pt &quot;Cantarell&quot;;</string>
    </property>
   </widget>
   <zorder>graphics_view</zorder>
   <zorder>seed_spin_box</zorder>
   <zorder>width_spin_box</zorder>
   <zorder>height_spin_box</zorder>
   <zorder>seed_label</zorder>
   <zorder>gameboard_width_label</zorder>
   <zorder>gameboard_height_label</zorder>
   <zorder>start_push_button</zorder>
   <zorder>reset_push_button</zorder>
   <zorder>quit_push_button</zorder>
   <zorder>game_widget</zorder>
   <zorder>timer_text_browser</zorder>
   <zorder>score_text_browser</zorder>
   <zorder>time_label</zorder>
   <zorder>score_lable</zorder>
   <zorder>game_info_text_browser</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quit_push_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>691</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
